src/gmail.js

```diff
@@ -3199,22 +3199,22 @@ var Gmail = function(localJQuery) {
         /**
            Retrieve the current "to" recipients
         */
-        to: function(to) {
-            return this.dom("to").val(to);
+        to: function() {
+            return this.dom("to").siblings('.vR').find('input').map((i, el) => el.value).get();
         },
 
         /**
            Retrieve the current "cc" recipients
         */
-        cc: function(cc) {
-            return this.dom("cc").val(cc);
+        cc: function() {
+            return this.dom("cc").siblings('.vR').find('input').map((i, el) => el.value).get();
         },
 
         /**
            Retrieve the current "bcc" recipients
         */
-        bcc: function(bcc) {
-            return this.dom("bcc").val(bcc);
+        bcc: function() {
+            return this.dom("bcc").siblings('.vR').find('input').map((i, el) => el.value).get();
         },
```

Comment:
These however.... I'd rather not have these as they fundamentally alter behaviour people may depend on.

Reply:
yes, people may depend on
but this methods don't work correctly and returns jquery object with empty textarea which is useless

we can change this method and keep the previous behavior, for example:
```javascript
bcc: function(arrayOfStrings) {
    return arrayOfStrings
        ? this.dom("bcc").siblings('.vR').find('input').map((i, el) => el.value).get()
        : this.dom("bcc");
}
```

Reply:
I've done some basic testing, and I have to admit the current implementation is not good. But your updated implementation only *retrieves* existing values and won't let a caller replace it, so it lacks a property this function used to have. So that's not all good either.

I certainly think this is something we should look into fixing properly, but I don't think this PR is the right place to do that.

Let's fix the type-annotations to match the implementation first, and *then* let's fix the implementation to act properly.

Sounds OK?

Reply:
yes, will update PR

Reply:
done